github-dark.user.css

```
@@ -29271,3 +29271,14 @@
     --color-tooltip-text: #c6c6c6;
   }
 }
+@-moz-document regexp("https?://github.com/sponsors") {
```

You may want to lock these changes from jumping sections

```suggestion
@-moz-document regexp("regexp('https?://github.com/sponsors(/.*)?')") {
```